src/Robo/Hooks/WebServerHook.php

```diff
@@ -3,13 +3,24 @@
 namespace Acquia\Blt\Robo\Hooks;
 
 use Acquia\Blt\Robo\BltTasks;
+use Acquia\Blt\Robo\Common\IO;
+use Acquia\Blt\Robo\Config\ConfigAwareTrait;
+use Acquia\Blt\Robo\Inspector\InspectorAwareTrait;
 use Consolidation\AnnotatedCommand\CommandData;
+use League\Container\ContainerAwareTrait;
```

Comment:
I don't understand where all of these includes are coming from. It doesn't seem like they are necessary.

Reply:
If you don't add these, a bunch of PHP warning are thrown.